examples/rdkafka_example.c

```diff
@@ -370,6 +395,31 @@ int main (int argc, char **argv) {
 		case 'A':
 			output = OUTPUT_RAW;
 			break;
+                case 'H':
+                {
+                        char *name, *val;
+                        size_t name_sz = -1;
+
+                        name = optarg;
+                        val = strchr(name, '=');
+                        if (val) {
+                                name_sz = (size_t)(val-name);
+                                val++; /* past the '=' */
+                        }
+
+                        if (!hdrs)
+                                hdrs = rd_kafka_headers_new(8);
+
+                        err = rd_kafka_header_add(hdrs, name, name_sz, val, -1);
```

Comment:
this is a bit tricky (dual meaning of the -1), but I can't think of anything I prefer and I guess it's ok.